int8  START_PATROL          = 0
int8 END_PATROL                = 1
int8 PATROL_LOSE_PILE  = 2

#patrol status
int32 type

#patrol name
string name